rpcclient/infrastructure.go

```diff
@@ -92,6 +92,10 @@ const (
 	// requestRetryInterval is the initial amount of time to wait in between
 	// retries when sending HTTP POST requests.
 	requestRetryInterval = time.Millisecond * 500
+
+	// defaultHttpTimeout is the default timeout for an http request, so the
+	// request does not block indefintily.
+	defaultHttpTimeout = time.Second * 2
```

Comment:
I think when the node is running on a Pi, certain I/O heavy calls can take a while, especially while processing blocks. Maybe 10 seconds instead? We do have retry logic already.

Reply:
good idea set to 10 sec.